ecs-cli/integ/e2e/fargate_service_test.go

```diff
@@ -41,19 +41,17 @@ func TestCreateClusterWithFargateService(t *testing.T) {
 	defer os.Remove(project.ComposeFileName)
 	defer os.Remove(project.ECSParamsFileName)
 
+	// Ensure cleanup of service and cluster
+	defer cmd.TestDown(t, conf)
+	defer cmd.TestServiceDown(t, project)
```

Comment:
the `defer` is called before `TestServiceUp` so I don't know how that will behave in case `ServiceUp` fails :P 

Should we move `defer cmd.TestDown(t, conf)` to l.36 and `defer cmd.TestServiceDown(t, project)` to l.50 instead?

Reply:
good call. fixed!